src/osx/textctrl_osx.cpp

```diff
@@ -206,19 +206,19 @@ wxSize wxTextCtrl::DoGetBestSize() const
         switch ( m_windowVariant )
         {
             case wxWINDOW_VARIANT_NORMAL :
-                hText = 22 - 6 ;
+                hText = 22 - 5 ;
```

Comment:
I think it would be nice to add `BORDER_SIZE` constant equal to 5, but I can do it myself when merging this, of course.

Reply:
Thanks for following this precisely I'm perfectly fine with that. I assume that you have verified that 5 lines can be displayed properly.

Reply:
Yes, I've tested wxTE_MULTILINE with 5 rows of text.